bucket/obs-plugin-droidcam.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.6.0",
+    "homepage": "https://www.dev47apps.com/obs/",
+    "description": "An OBS plugin that allows the user to use their phone as a webcam for OBS Studio.",
+    "license": "GPL-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/dev47apps/droidcam-obs-plugin/releases/download/1.6.0/DroidCamOBS.Setup.1.6.0.exe#/dl.7z",
+            "hash": "32383f797337464a1416116726b2cc251a39a459765ab2e5532f247a4ee19d3c"
+        }
+    },
+    "extract_dir": "$_13_",
+    "pre_install": [
+        "Remove-Item \"$dir\\data\\obs-plugins\\droidcam-obs\\`$_13_\" -Recurse",
+        "if (Test-Path \"$(appdir obs-studio $global)\") {",
+        "   Copy-Item \"$dir\\data\\obs-plugins\\droidcam-obs\" \"$(appdir obs-studio $global)\\current\\data\\droidcam-obs\" -Recurse -ErrorAction 'SilentlyContinue'",
+        "   Copy-Item \"$dir\\obs-plugins\\64bit\\*\" \"$(appdir obs-studio $global)\\current\\obs-plugins\\64bit\" -Recurse -ErrorAction 'SilentlyContinue'",
+        "}",
+        "elseif (!(Test-Path \"$(appdir obs-studio $global)\")) {error \"$app requires 'obs-studio' to be installed\"; break}"

```

Comment:
we should use `depends`, even if does not consider other variant versions. but we are considering them now, so just drop this line.